### PR TITLE
review the code

### PR DESCRIPTION
**Pull request by ish4qq**
please review the code for the branch v2, thanks.

### CODE REVIEW

File: Messaging component [hunk 1]
```diff
@@ -15,3 +15,13 @@ System.out.println("Message will be recieved by: " + bsc_Reciever);
 System.out.println("The message content: " + bsc_Info);
 }
 }
+
+class main {
+public static void main(String[] args) {
+Message m=new Message();
+
+m.setValues("Ishaq", "Sulaiman", "Hello, my name is Ishaq.");   //changed Message to setValues
```
Comment (ish4q) [inline Messaging component]: Changes value from "Ishaq" to "Ahmed".

**Review by ish4q — CHANGES_REQUESTED** [review]
please check the comment to see the changes required

File: Messaging component (hunk 1 above)
Comment (ish4qq) [reply Messaging component]: Please check now.

**Review by ish4q — APPROVED** [review]
The change request has been solved. Approved.

**Comment by ish4q** [comment]
Codebase looking good so far. Closing request.

**Pull request closed without merging**